README.md

```diff
@@ -11,7 +11,12 @@ You can find some basic setup instructions and links to the Standard OSS project
 ### Installation
 
 1. If you haven't already, [install the Code Climate CLI](https://github.com/codeclimate/codeclimate).
-2. Add the engine and enable it in your `.codeclimate.yml` file.
+2. Enable the engine by adding the following under `plugins` in your `.codeclimate.yaml`:
```

Comment:
adjusting README to mimic the rubocop engine instructions (which this repo was apparently made from since it has some of the original commits in it). Modified slightly to use `standard` as the engine name below not `rubocop` seen [here](https://github.com/codeclimate/codeclimate-rubocop/blob/master/README.md)

This also sets it up nicely for when the next standard channel (> 1.37.0) is released which will require a minimum ruby version of >= 3. I suspect given the default implementation is to either not set the channel explicitly in the .codeclimate.yml file _or_ set it to "latest", we may (at that time in the future) see some regression when we move to that next version. 

The reason I think having this setup in the README helps is we can add a `channel:` line to it with some comments about which explicit channel to use if you are running ruby < 3 since "latest" will be at that point running the standard version that requires it